cmd/neofs-cli/modules/container.go

```diff
@@ -1,23 +1,210 @@
 package cmd
 
 import (
+	"context"
+	"crypto/sha256"
+	"errors"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"time"
 
+	"github.com/google/uuid"
+	"github.com/mr-tron/base58"
+	"github.com/nspcc-dev/neofs-api-go/pkg/acl"
+	"github.com/nspcc-dev/neofs-api-go/pkg/container"
+	"github.com/nspcc-dev/neofs-api-go/pkg/netmap"
+	"github.com/nspcc-dev/neofs-api-go/pkg/owner"
+	v2container "github.com/nspcc-dev/neofs-api-go/v2/container"
+	"github.com/nspcc-dev/neofs-node/pkg/policy"
 	"github.com/spf13/cobra"
 )
 
+const (
+	attributeDelimiter = ":"
+
+	awaitTimeout = 120 // in seconds
+)
+
+var (
+	containerOwner string
+
+	containerACL        string
+	containerNonce      string
+	containerPolicy     string
+	containerAttributes []string
+	containerAwait      bool
+
+	containerID string
+)
+
 // containerCmd represents the container command
 var containerCmd = &cobra.Command{
 	Use:   "container",
-	Short: "Operations with Containers",
-	Long:  `Operations with Containers`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("container called")
+	Short: "Operations with containers",
+	Long:  "Operations with containers",
+}
+
+var listContainersCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List all created containers",
+	Long:  "List all created containers",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var (
+			response []*container.ID
+			oid      *owner.ID
+			err      error
+
+			ctx = context.Background()
+		)
+
+		cli, err := getSDKClient()
+		if err != nil {
+			return err
+		}
+
+		switch containerOwner {
+		case "":
+			response, err = cli.ListSelfContainers(ctx)
+		default:
+			oid, err = ownerFromString(containerOwner)
+			if err != nil {
+				return err
+			}
+
+			response, err = cli.ListContainers(ctx, oid)
+		}
+
+		if err != nil {
+			return fmt.Errorf("rpc error: %w", err)
+		}
+
+		// print to stdout
+		prettyPrintContainerList(response)
+
+		return nil
+	},
+}
+
+var createContainerCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create new container",
+	Long: `Create new container and register it in the NeoFS. 
+It will be stored in sidechain when inner ring will accept it.`,
```

Comment:
It will be stored in sidechain when inner ring accepts it

Reply:
Fixed.